.github/workflows/extension.yml

```
@@ -0,0 +1,68 @@
+name: Publish devtools extension
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    branches:
+      - main
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    if: ${{ contains(github.event.head_commit.message, 'applying package updates') || github.event_name == 'workflow_dispatch' }}
+
+    outputs:
+      status: ${{ steps.verify-extension-changed.outputs.any_changed }}
+
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout [main]
+
+      - name: Verify extension has published
```

```suggestion
      - name: Verify extension has changed
```

This job really doesn't check the published status of the extension at all